### PR TITLE
Remove legacy examples

### PR DESCRIPTION
**Pull request by xzilja**

### CODE REVIEW

File: servers/relay/test/http.spec.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import "mocha";
 import axios, { AxiosInstance } from "axios";
-import { use, expect } from "chai";
+import { expect, use } from "chai";
 import chaiHttp from "chai-http";
 import { Agent } from "https";
-
+import "mocha";
```
Comment (bkrem) [inline servers/relay/test/http.spec.ts]: This seems like an unrelated change, was this intentional?

**Review by bkrem — APPROVED** [review]
🧹 ✨

File: servers/relay/test/http.spec.ts (hunk 1 above)
Comment (xzilja) [reply servers/relay/test/http.spec.ts]: prettier slipped through when I visited this file

**Review by pedrouid — APPROVED** [review]
👍
